test/features/pagecontent/redirects.js

```diff
@@ -208,7 +208,7 @@ describe('Redirects', function() {
         })
         .then(function(res) {
             assert.deepEqual(res.status, 302);
-            assert.deepEqual(res.headers.location, 'Main_Page');
+            assert.deepEqual(res.headers.location, 'Main_Page#Test');
```

Comment:
Would be good to add a test for a hashbang containing %23 and also for query order.

Reply:
Done